README.md

```diff
@@ -63,13 +63,19 @@ rga will search for all binaries it calls in \$PATH and the directory itself is
 
 ### Windows
 
-Install ripgrep-all via [Chocolatey](https://chocolatey.org/packages/ripgrep-all):
+#### Chocolatey
 
 ```
 choco install ripgrep-all
 ```
 
-Note that installing via chocolatey or scoop is the only supported download method. If you download the binary from releases manually, you will not get the dependencies (for example pdftotext from poppler).
+#### Scoop
+
+```
+scoop install rga
+```
+
+Note that installing via [chocolatey](https://chocolatey.org/packages/ripgrep-all) or [scoop](https://github.com/ScoopInstaller/Main/blob/master/bucket/rga.json) is the only supported download method. If you download the binary from releases manually, you will not get the dependencies (for example pdftotext from poppler).
```

Comment:
This text (and the next paragraph) should go above the "Chocolatey" heading. Otherwise the text will appear to be part of the "Scoop" section when it's really applicable to the entire "Windows" section.

Reply:
Fixed, thank you for correction.